### PR TITLE
Add CWS and CSPM 7.50 release notes

### PR DESCRIPTION
**Pull request by lebauce**
### What does this PR do?

Add 7.50 release notes for CSPM and CWS.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: releasenotes/notes/csm-7-50-14413f70a64656f9.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+features:
+  - |
+    CWS: Ability to send a signal to a process when a rule was triggered.
+    CWS: Add Kubernetes user session context to events, in particular the username, UID and groups of the user that ran the commands remotely.
+
+enhancements:
+  - |
+    CWS: Support for Ubuntu 23.10.
+    CWS: Support using TCP in addition in UNIX sockets.
```
Comment (paulcacheux) [inline releasenotes/notes/csm-7-50-14413f70a64656f9.yaml:10]: in addition "to" maybe ? and maybe explain a bit what we mean by support ? I don't think it's super clear

File: releasenotes/notes/csm-7-50-14413f70a64656f9.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    CWS: Reduce memory usage of ring buffer on machines with large memory capacities.
```
Comment (YoannGh) [inline releasenotes/notes/csm-7-50-14413f70a64656f9.yaml:11]:
```suggestion
    CWS: Reduce memory usage of ring buffer on machines with more than 64 CPU cores.
```

File: releasenotes/notes/csm-7-50-14413f70a64656f9.yaml (hunk 1 above)
Comment (lebauce) [reply releasenotes/notes/csm-7-50-14413f70a64656f9.yaml]: Indeed. I completely removed it from the change log

File: releasenotes/notes/csm-7-50-14413f70a64656f9.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+features:
+  - |
+    CWS: Ability to send a signal to a process when a rule was triggered.
+    CWS: Add Kubernetes user session context to events, in particular the username, UID and groups of the user that ran the commands remotely.
+    CSPM: Add a system-probe module to run some compliance checks (currently disabled by default).
+
+enhancements:
+  - |
+    CWS: Support for Ubuntu 23.10.
+    CWS: Reduce memory usage of ring buffer on machines with more than 16 CPU cores.
```
Comment (paulcacheux) [inline releasenotes/notes/csm-7-50-14413f70a64656f9.yaml:11]:
```suggestion
    CWS: Reduce memory usage of ring buffer on machines with more than 64 CPU cores.
```

File: releasenotes/notes/csm-7-50-14413f70a64656f9.yaml (hunk 2 above)
Comment (lebauce) [reply releasenotes/notes/csm-7-50-14413f70a64656f9.yaml]: Good point. I think it's 16 CPU cores though

Comment (YoannGh) [reply releasenotes/notes/csm-7-50-14413f70a64656f9.yaml]: The memory usage for the ]16;64] bucket didn't change, it's still `16 * 256 * os.Getpagesize()` https://github.com/DataDog/datadog-agent/pull/19940/files

Comment (paulcacheux) [reply releasenotes/notes/csm-7-50-14413f70a64656f9.yaml]: Yes it's only 64 and more that changed

Comment (lebauce) [reply releasenotes/notes/csm-7-50-14413f70a64656f9.yaml]: Oh right, I read too fast

**Review by 0intro — APPROVED** [review]

**Review by paulcacheux — APPROVED** [review]

**Review by YoannGh — APPROVED** [review]

**Review by git-thuerk-done — APPROVED** [review]
Looks good.

**Pull request merged**